src/components/modals/KPIProgressModal.vue

```diff
@@ -4,6 +4,17 @@
       {{ $t(record ? 'kpi.editValue' : 'kpi.newValue') }}
     </template>
 
+    <i18n path="kpi.help.apiProgress" tag="p">
+      <template #apiLink>
+        <router-link :to="{ name: 'Api' }" target="_blank">
+          <span>{{ $t('general.api') }}</span>
+        </router-link>
+      </template>
+    </i18n>
+    <ProgressUpdateAPIExample :path="`kpi/${activeKpi.id}`" />
+    <hr class="ods-hr" />
+    <p class="help-text">{{ $t('kpi.help.manualProgress') }}</p>
```

Comment:
Bør vi sette eksemplet nederst? Man åpner tross alt modalen for å legge til noe manuelt? Og ikke vise eksemplet, eller vise annet kall (`PUT <date>`), når man velger "Endre målepunkt"?

Reply:
Hmm, ble det litt rart det også?
![image](https://user-images.githubusercontent.com/3673262/207312381-79c06bdb-ff22-4d5b-84d0-885a9187c76e.png)

Reply:
Hva med en info-boks under "Lagre"-knappen? F.eks. [ODS "informative alert"](https://designsystem.oslo.kommune.no/latest/komponenter/alert/index)? Eller kanskje en oslo-dark-blue boks med hvit tekst (lit ala revamped tooltip)?

Reply:
Tester det.

Reply:
Ble ganske mye bedre med å bare flytte det under lagreknappen.